spec/models/proceeding_type_spec.rb

```diff
@@ -60,4 +60,11 @@
       end
     end
   end
+
```

Comment:
Not sure if this belongs in this pr?

Reply:
Originally on the to-do list were three fixes, as I added the code to ProceedingType the day before yesterday, it was a simpler fix for me than the other two! 😄